gluten-substrait/src/main/java/org/apache/gluten/substrait/rel/ReadRelNode.java

```diff
@@ -51,6 +52,10 @@ public class ReadRelNode implements RelNode, Serializable {
     this.extensionNode = extensionNode;
   }
 
+  public void setStreamKafka(boolean streamKafka) {
+    this.streamKafka = streamKafka;
+  }
```

Comment:
This is like a workaround given `ReadRelNode` was likely designed to be immutable. Could you do some refactors to make `streamKafka` final? Thanks.